src/wrapper/jnienv.rs

```diff
@@ -690,7 +690,7 @@ impl<'a> JNIEnv<'a> {
                 )
                 .into(),
                 Primitive::Void => {
-                    jni_non_void_call!(
```

Comment:
Thanks! It must have been the only such usage — the other already use jni_void_call.